bro_connector/main/management/commands/frd_sync_to_bro.py

```diff
@@ -1,13 +1,396 @@
+import os
+import time
+
 from django.core.management.base import BaseCommand
+from bro_exchange.broxml.frd.requests import FrdStartregistrationTool
+from bro_exchange.bhp.connector import (
+    validate_sourcedoc,
+    upload_sourcedocs_from_dict,
+    check_delivery_status,
+)
+from frd.models import FormationResistanceDossier, FrdSyncLog
+from datetime import datetime
+from main.settings.base import FRD_SETTINGS
+
+#############
+## Command ##
+#############
 
 
 class Command(BaseCommand):
     """
     Handles the sync between the bro-connector and the BRO.
     Runs daily as cronjob.
     """
-    help = 'Syncs the FRD data to the BRO'
+
+    help = "Syncs the FRD data to the BRO"
 
     def handle(self, *args, **kwargs):
+        self.handle_startregistrations()
+        self.handle_measurements()
+
+    def handle_startregistrations(self):
+        """
+        Handles the startregistrations for all FRD's without bro_id.
+        For each new FRD, the FrdStartregistration class is used to hande the registration
+        """
+        new_dossiers_qs = self.get_new_dossiers()
+
+        if len(new_dossiers_qs) != 0:
+            for new_dossier in new_dossiers_qs:
```

Comment:
Je kan gewoon over de queryset heen loopen, als 'ie leeg is doet 'ie niks. Dus de check of 'ie leeg is is niet nodig.

```suggestion
        for frd in self.get_new_dossiers():
```

Je kan `new_dossier` gebruiken, maar elders gebruik je overal `frd`, dat is misschien dan handiger voor de herkenbaarheid. Ik hou niet zoveel van afkortingen, maar `formation_resistance_dossier` is wel wat lang. Ik weet niet hoeveel type dossiers er zijn/komen? `fr_dossier`? 

't Moet wel consistent gebruikt worden.